src/librustc/ty/util.rs

```diff
@@ -613,36 +612,6 @@ impl<'a, 'gcx, 'tcx> TyCtxt<'a, 'gcx, 'tcx> {
         })
     }
 
-    /// Returns `true` if the node pointed to by `def_id` is a static item, and its mutability.
-    pub fn is_static(&self, def_id: DefId) -> Option<hir::Mutability> {
```

Comment:
What if we keep this as `-> bool` and make the query something like `static_mutability`?
Instead of having the functionality of both in one?